src/scss/components/eyebrow/index.scss

```diff
@@ -0,0 +1,126 @@
+// ----- BASE ELEMENTS ---------------
+
+.eyebrow {
+  overflow: hidden;
+  width: 100%;
+  height: 56px;
```

Comment:
### Please update ALL pixel values to rem values:

```
1 rem = 16 pixels
```

Reply:
Changes have been done, Please review